src/components/SignIn.tsx

```diff
@@ -202,7 +203,7 @@ const web3AuthClientId = process.env.REACT_APP_WEB3AUTH_CLIENT_ID as string;
 type LOGIN_PROVIDER_TYPE = 'google' | 'facebook' | 'apple' | 'discord' | 'twitch' | 'github' | 'twitter' | 'email_passwordless'
 
 interface SignInProps {
-  onWeb3ProviderSet: (web3Provider: any) => void;
+  onWeb3ProviderSet: (web3Provider: any, isWagmi?: boolean) => void;
```

Comment:
any particular reason why we need `isWagmi`? trying to read from code and it seems it can just go with wagmi bu default, correct me if I'm wrong

Reply:
`setConnectedProvider(isWagmi ? web3.currentProvider.provider : web3.currentProvider)` I'm doing this in the , so here I've done the login through wallets using wagmi not the social logins

Reply:
isn't `web3.currentProvider.provider` === `web3.currentProvider`?

Reply:
no in case of wagmi it's `web3.currentProvider.provider`